js8call_cmake.patch

```diff
@@ -0,0 +1,26 @@
+--- CMakeLists.txt.orig	2023-06-30 13:34:16.937165343 +0200
++++ CMakeLists.txt	2023-06-30 13:44:23.048494595 +0200
+@@ -1067,12 +1067,12 @@
+   # menu with an icon
+   install (
+     FILES js8call.desktop
+-    DESTINATION /usr/share/applications
++    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications
+     #COMPONENT runtime
+     )
+   install (
+     FILES icons/Unix/js8call_icon.png
+-    DESTINATION /usr/share/pixmaps
++    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/128x128/apps
+     #COMPONENT runtime
+     )
+ 
+@@ -1285,7 +1285,7 @@
+ 
+ set (CPACK_RPM_PACKAGE_ARCHITECTURE ${CMAKE_SYSTEM_PROCESSOR})
+ set (CPACK_RPM_PACKAGE_REQUIRES "qt5-qtserialport >= 5.2, qt5-qtmultimedia >= 5.2, qt5-qtsvg, libusb, systemd-udev, glibc >= 2, libgfortran >= 8.4.0")
+-set (CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION /usr/share/pixmaps /usr/share/applications /usr/share/man /usr/share/man1)
++set (CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION /usr/share/icons/hicolor/128x128/apps /usr/share/applications /usr/share/man /usr/share/man1)
```

Comment:
I'm pretty sure should be CMAKE_INSTALL_PREFIX here instead of `/usr`


Also please submitt the patch upstream.

Reply:
I wanted to keep the patch minimal, but you are right.